components/ImageCard.vue

```diff
@@ -0,0 +1,38 @@
+<template>
+  <v-card class="ImageCard" :href="url">
+    <div class="ImageContent">
+      <v-img :src="imgpath"></v-img>
```

Comment:
```suggestion
      <v-img :src="imgpath" />
```
こうすることで少し綺麗に、コンパクトにかけると思います。